lms/djangoapps/verify_student/views.py

```diff
@@ -724,7 +731,7 @@ def create_order(request):
         return HttpResponseBadRequest(_("No selected price or selected price is below minimum."))
 
     if current_mode.sku:
-        return create_order_with_ecommerce_service(request.user, course_id, current_mode)
+        return create_order_with_ecommerce_service(request.user, course_id, current_mode, request.POST.get('processor'))
```

Comment:
@wedaly this should only happen for the stale cached js right?  I'm changing it to default to `settings.CC_PROCESSOR_NAME` as a fallback.